### ISSUE TITLE
Fatal error - B9PartSwitch.TankType and .TankResource for LiquidFuel

### ISSUE THREAD

**Issue by als0052** [blowfishpro/B9PartSwitch#217]
I'm getting a strange fatal error thrown by this mod. I've tried all the trouble shooting I can to no avail. When I remove B9PartSwitch I can load the game, but adding it through CKAN causes the fatal exception. Running KSP 1.10.1 and all mods installed via CKAN. Log attached in the .zip file. Let me know if I need to attach anything else. 

![ExceptionImage](https://user-images.githubusercontent.com/43427976/117085141-996ae900-ad0e-11eb-807c-0b62d6c6ea38.png)
[B9PS_FatalException.zip](https://github.com/blowfishpro/B9PartSwitch/files/6424546/B9PS_FatalException.zip)

**Comment by als0052** [comment]
I just found out you can export entire mod lists to .txt files. Attaching that as well just because. 

[KSP_1.10.1.2939_modlist_05-04-2021_20-28pm.txt](https://github.com/blowfishpro/B9PartSwitch/files/6424559/KSP_1.10.1.2939_modlist_05-04-2021_20-28pm.txt)

**Comment by blowfishpro** [comment]
Your GameData/Squad folder is missing things that the core game needs.  Please reinstall it.

**Comment by als0052** [comment]
> Your GameData/Squad folder is missing things that the core game needs. Please reinstall it.

Already tried that

**Comment by blowfishpro** [comment]
The stock resources are missing from your install.  This isn't a B9PartSwitch issue, it's just the only thing the complains loudly ranter than randomly breaking later in-game.

**Comment by als0052** [comment]
> The stock resources are missing from your install. This isn't a B9PartSwitch issue, it's just the only thing the complains loudly ranter than randomly breaking later in-game.

Well I don't know what happened but I gave reinstalling (re-re-install?) another shot. After re-re-installing this second time it seems to be working. Not sure what I did differently the first re-install attempt lol.  Thanks for at least having proper error logging in your mods! And the quick response. No idea how you actually interpret those log files but its all greek to me.
